### ISSUE TITLE
sharptimer_trigger_push_fix

### ISSUE THREAD

**Issue by NordicGamerFE** [Letaryat/poor-sharptimer#31]
sharptimer_trigger_push_fix does not actually fix push triggers and actually breaks them.

**Comment by NordicGamerFE** [comment]
i dont use sharptimer_trigger_push_fix but i did once before and what I noticed was that every map that had a working jumppad suddenly didn't

**Comment by exkludera** [comment]
> please enable debug mode and let me know what it spits out upon entering a trigger_push zone

```
06:03:49 [INFO] (plugin:SharpTimer) [SharpTimerDebug] trigger_push fix: Player velocity adjusted for test by 5.0154037
06:03:50 [INFO] (plugin:SharpTimer) [SharpTimerDebug] trigger_push OnStartTouch Player velocity adjusted for test by 35
```

tested on mg_minecraft_aphixed where player should gain speed, works without the sharptimer_trigger_push_fix setting and breaks with it on😞

**Comment by M-archand** [comment]
Feature was removed from SharpTimer and moved to [STFixes](https://github.com/rcnoob/STFixes).
